inc/c_util/rc_ptr.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef RC_PTR_H
+#define RC_PTR_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
```

Comment:
>stddef [](http://example.com/codeflow?start=10&length=6)

where is this header needed?

Reply:
I thought we include this if we use NULL